backend/golf/settings.py

```diff
@@ -0,0 +1,124 @@
+"""
+Django settings for golf project.
+
+Generated by 'django-admin startproject' using Django 2.2.6.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.2/ref/settings/
+"""
+
```

Comment:
You're goint to want some kind of variables to track the environment this is running in